vue-app/src/views/Join.vue

```diff
@@ -84,17 +84,16 @@
                   <p
                     :class="{
                       error: true,
-                      hidden: !$v.form.project.tagline.$error,
+                      hidden: !$v.form.project.tagline.$error || !$v.form.project.tagline.$model,
```

Comment:
Was thinking maybe we could use the maxLength attribute on $v.form.project.tagline, but accomplishes the same thing. This would change to:
` || $v.form.project.tagline.maxLength` as we would want it to be hidden if its under the maxLength (maxLength === true if $v.form.project.tagline.$model.len <= $v.form.project.tagling.maxLength). That way if its over maxLength then it will be false, and no longer be hidden. Might just be a bit more descriptive of a conditional is all, but this obviously works as is already too as its more or less checking the same thing.